extensions/amp-story/1.0/amp-story-desktop.css

```diff
@@ -132,15 +132,32 @@ amp-story[standalone][desktop] {
   background: linear-gradient(to right, rgba(33,33,33,0) 0%, rgba(33,33,33,0.32) 100%)!important;
 }
 
+[dir=rtl] [desktop] > .next-container::before {
+  right: auto !important;
+  left: 0 !important;
+  background: linear-gradient(to right, rgba(33,33,33,0.32) 0%, rgba(33,33,33,0) 100%)!important;
+}
+
```

Comment:
For all these new `rtl` rules, can we reuse the existing CSS and simply add new selectors? For example, I think these properties apply to both:
- `prev-container` if `ltr`
- `next-container` if `rtl`

Which could reduce code duplication, and give something like:

````
[desktop] > .prev-container::before,
[dir=rtl] [desktop] > .next-container::before {
  left: 0 !important;
  right: auto !important;
  background: linear-gradient(to right, rgba(33,33,33,0.32) 0%, rgba(33,33,33,0) 100%)!important;
}
````